### ISSUE TITLE
`nuxt-module-build` bin command error

### ISSUE THREAD

**Issue by benjamincanac** [nuxt/module-builder#17]
I have this error when running yarn build with `nuxt-module-build`:

```
/bin/sh: /Users/benjamincanac/Projects/nuxt/strapi-module/node_modules/.bin/nuxt-module-build: /bin/env: bad interpreter: No such file or directory
```

I might not have use it well, so I made a PR on the strapi-module: https://github.com/nuxt-community/strapi-module/pull/214

**Comment by pi0** [comment]
Thanks for issue. Bin issue should be fixed in 0.1.3. (#14)

**Comment by benjamincanac** [comment]
Working perfectly, thanks!
